### PR TITLE
Build Layout of detail user

### PR DESCRIPTION
**Pull request by ghost**
## Description
 Buil layout of detail user
## Reference Requirements
- Product Backlog ID: PBI-1
- Task ID: #2.3

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

- routes/web.php
- resources/views/backend/
- app/Http/Controllers/

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -12,5 +12,8 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('backend.layouts.main');
```
Comment (at-luanle) [inline routes/web.php]: nen de vao Controller

File: routes/web.php [hunk 2]
```diff
@@ -12,5 +12,8 @@
 */
 
 Route::get('/', function () {
```
Comment (at-luanle) [inline routes/web.php]: phan nay cua phan FONT-END, nen khong can

File: routes/web.php [hunk 3]
```diff
@@ -12,5 +12,8 @@
...
-    return view('welcome');
+    return view('backend.layouts.main');
 });
+Route::get('admin/user/detail', function () {
+    return view('backend.users.detail');
```
Comment (at-luanle) [inline routes/web.php]: Em nen de vao Controller User

File: routes/web.php (hunk 1 above)
Comment (at-nghiavo) [reply routes/web.php]: em đã fix

File: routes/web.php (hunk 2 above)
Comment (at-nghiavo) [reply routes/web.php]: function của master đó a

File: app/Http/Controllers/HomeController.php [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Routing\Controller as BaseController;
+
+class HomeController extends BaseController
```
Comment (at-luanle) [inline app/Http/Controllers/HomeController.php]: em dang lam User thi nen lam UserController

File: routes/web.php [hunk 5]
```diff
@@ -14,6 +14,4 @@
 Route::get('/', function () {
     return view('backend.layouts.main');
 });
-Route::get('admin/user/detail', function () {
-    return view('backend.users.detail');
-});
\ No newline at end of file
+Route::get('admin/user/detail', 'HomeController@showDetails');
```
Comment (at-luanle) [inline routes/web.php]: em nen sua lai UserController

File: app/Http/Controllers/HomeController.php (hunk 4 above)
Comment (at-nghiavo) [reply app/Http/Controllers/HomeController.php]: đã fix nha a

**Pull request closed without merging**